### PR TITLE
Unk Debug Menu 3 and Start Menu Debug

### PR DESCRIPTION
**Pull request by PikalaxALT**

### CODE REVIEW

File: src/debug/unk_debug_menu_3.c [hunk 1]
```diff
@@ -215,192 +216,64 @@ size_t debug_sub_813C740(u8 * dest)
...
+    u8 * end = gUnknown_Debug_845E443End;
+    struct Pokemon *mon = gEnemyParty;
+    size_t size = end - src;
+
+    memcpy(dest, src, size);
+    debug_sub_813C638(mon, SPECIES_UNOWN, 21, ITEM_DREAM_MAIL);
+    debug_sub_813C6AC(mon, &mail);
+    mon->box.checksum = 0;
+
+    // nonmatching pointer arithmetic
```
Comment (ProjectRevoTPP) [inline src/debug/unk_debug_menu_3.c]: but that one actually did ~~solve petscop~~ match debug_sub_813C888

Comment (PikalaxALT) [reply src/debug/unk_debug_menu_3.c]: yes, lemme remove that comment

File: src/debug/unk_debug_menu_3.c [hunk 2]
```diff
@@ -0,0 +1,362 @@
...
+#if DEBUG
+
+extern u8 gUnknown_Debug_845DAE1[];
+extern u8 gUnknown_Debug_845DAE1End[];
+
+size_t debug_sub_813C404(u8 * dest)
+{
+    size_t size = gUnknown_Debug_845DAE1End - gUnknown_Debug_845DAE1;
+    memcpy(dest, gUnknown_Debug_845DAE1, size);
+    unref_sub_812620C(dest, gUnknown_Debug_845DAE1);
```
Comment (huderlem) [inline src/debug/unk_debug_menu_3.c]: Looks like there are a bunch of calls to `unref` subs.  Might as well remove that prefix or replace with `debug`.

File: src/debug/unk_debug_menu_3.c [hunk 3]
```diff
@@ -0,0 +1,362 @@
+#include "global.h"
```
Comment (huderlem) [inline src/debug/unk_debug_menu_3.c]: Looks like `unk_debug_menu_3.c` should be named `mystery_event_debug_menu.c`

File: data/debug_menu_3.s [hunk 4]
```diff
@@ -1,106 +1,311 @@
 .if DEBUG
```
Comment (huderlem) [inline data/debug_menu_3.s]: Similarly, this can be `data/mystery_event_debug_menu.s`

File: include/macros/me_script.inc [hunk 5]
```diff
@@ -0,0 +1,88 @@
+	.macro me_nop
```
Comment (huderlem) [inline include/macros/me_script.inc]: I'd prefer the filename to be verbose: `mystery_event_script.inc`.  The macro names are fine abbreviated.

**Pull request merged**